catalog/controller/extension/payment/securesubmit.php

```diff
@@ -461,6 +461,12 @@ private function sessionVar($key = null, $val = null)
          */
         $HPS_KEY = $this->securesubmit_fraud_fail_var;
```

Comment:
shall we add a condition here to make sure $this->securesubmit_fraud_fail_var is not empty?